react/Empty/Readme.md

```diff
@@ -31,10 +31,9 @@ const styles = {
   }
 };
 
-<div style={styles.empty}>
+<div style={styles.empty} id='empty'>
```

Comment:
OK, this is why we don't have the id in the snapshot. 

We should not attach an id to this div but to the `Empty` component below ;)

Reply:
Not resolved...